crispy_forms_materialize/templates/materialize_css_forms/field.html

```diff
@@ -3,25 +3,42 @@
 {% if field.is_hidden %}
     {{ field }}
 {% else %}
-    <div id="div_{{ field.auto_id }}" class="input-field {% if field.errors and form_show_errors %} error{% endif %}{% if field|is_checkbox %} checkbox{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}">{% spaceless %}
+    {% if field|is_checkbox %}
+    <{% if tag %}{{ tag }}{% else %}p{% endif %}>
+    {% elif field|is_checkboxselectmultiple %}
```

Comment:
Hi @mmahkamov , thanks for your contributions! A quick question about this line, why there is no definition for 'field|is_checkboxselectmultiple' ? Before your changes it was:
```
        {% if field|is_checkboxselectmultiple %}
            {% include 'materialize_css_forms/layout/checkboxselectmultiple.html' %}
        {% endif %}
```

And now its:

```
    {% if field|is_checkbox %}
        <{% if tag %}{{ tag }}{% else %}p{% endif %}>
    {% elif field|is_checkboxselectmultiple %}

    {% else %}
```

Sorry, i dont understand those changes, could you please explain me a bit? thx!

Reply:
I was not sure if a custom tag should be handled in is_checkboxselectmultiple case. The empty case is there just as a reminder. Can be safely removed.

Reply:
The checkboxselectmultiple.html template is included further down